python-package/securexgboost/core.py

```diff
@@ -401,16 +404,12 @@ class DMatrix(object):
     def __init__(self, data_dict, encrypted=True, silent=False,
             feature_names=None, feature_types=None): 
         """
-        Load a DMatrix from encrypted files at the enclave server, where
-        each file is encrypted with a particular user's symmetric key.
-
         Parameters
         ----------
-        data_dict : dictionary 
-            Keys: Usernames
-            Values: Path to training data of corresponding user. Must be the absolute path.
+        data_dict : dict [str, str]
```

Comment:
Will Sphinx properly generate this in the documentation? I haven't seen the [str, str] before. If Sphinx can parse this properly, maybe we can change `[str, str]` to `{str: str}`.

If this doesn't work, looks like you can do something like 
```
data_dict : dict
    <Description of keys and values>
```

Reply:
Yeah this worked, but changing it to `dict, {str: str}` which reads better.